api/HTMLInputElement.json

```diff
@@ -543,7 +543,7 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "3"
+              "version_added": "3.6"
```

Comment:
I recently found out that my Firefox 3 VM was of version 3.6.29, rather than a 3.0.x release for some odd reason.  Since that PR, I fixed that VM, but I didn't realize I submitted any PRs with data involving that version.  In other words, any changes from 3 to 3.5/3.6 from my own PRs should be determined as the more accurate data.

Reply:
OK, that explains it! As discussed in another PR, if you could fix 3 vs. 3.6 issues in a separate PR, that would be great!